cardano-cli/test/cli/genesis-create/data/genesis.spec.json

```diff
@@ -37,6 +37,7 @@
             "minor": 0
         },
         "rho": 0,
-        "tau": 0
+        "tau": 0,
+        "minUTxOValue": 0
```

Comment:
I thought we were not going to change this, since we made it optional in the parser.